images/base/Dockerfile

```diff
@@ -122,7 +122,7 @@ RUN eval "$(gimme "${GO_VERSION}")" \
 # stage for building containerd
 FROM go-build as build-containerd
 ARG TARGETARCH GO_VERSION
-ARG CONTAINERD_VERSION="v1.7.15"
+ARG CONTAINERD_VERSION="v1.7.16"
```

Comment:
any notable changes to be aware of?

we also usually see runc changed at the same time

Reply:
I didn't see any breaking changes, just bug fixes and a CVE fix.

runc has a new RC tag [`v1.2.0-rc.1`](https://github.com/opencontainers/runc/releases/tag/v1.2.0-rc.1) but this is a minor version upgrade from what Kind uses currently, which is `v1.1.12`, so I was wary about upgrading to that one.